Babylon/commands/api/scenarioruns/cumulated_logs.py

```diff
@@ -0,0 +1,36 @@
+from logging import getLogger
+from typing import Any
+
+from click import command, option
+
+from Babylon.commands.api.scenarioruns.service.api import ScenarioRunService
+from Babylon.utils.credentials import pass_azure_token
+from Babylon.utils.decorators import timing_decorator, wrapcontext, retrieve_state
+from Babylon.utils.response import CommandResponse
+from Babylon.utils.typing import QueryType
+
+logger = getLogger("Babylon")
+
+
+@command()
+@wrapcontext()
+@timing_decorator
+@pass_azure_token("csm_api")
+@option("--org-id", "organization_id", type=QueryType())
```

Comment:
i think `QueryType` will be deprecated with the new state feat. For now, you can replace it by `str` stp